docs/agent-bridging/ref/findInstances.md

```diff
@@ -0,0 +1,194 @@
+        }
+    },
+    "meta": {
+        "requestUuid": "<requestUuid>",
+        "timestamp": "2020-03-...",
+        "destination": { "desktopAgent": "agent-B"}, //destination agent
+        "source": {
+            "appId": "AChatApp",
+            "instanceId": "02e575aa-4c3a-4b66-acad-155073be21f6"
+        }
+    }
+}
+```
+
+The Desktop Agent Bridge should only forward the request to the requested Desktop Agent and handle the message exchange as a **Request Response (single)**.
+
+## Response format
+
+### Response message schemas
+
+- [https://fdc3.finos.org/schemas/next/bridging/findInstancesAgentResponse.schema.json](/schemas/next/bridging/findInstancesAgentResponse.schema.json)
+- [https://fdc3.finos.org/schemas/next/bridging/findInstancesAgentErrorResponse.schema.json](/schemas/next/bridging/findInstancesAgentErrorResponse.schema.json)
+- [https://fdc3.finos.org/schemas/next/bridging/findInstancesBridgeResponse.schema.json](/schemas/next/bridging/findInstancesBridgeResponse.schema.json)
+- [https://fdc3.finos.org/schemas/next/bridging/findInstancesBridgeErrorResponse.schema.json](/schemas/next/bridging/findInstancesBridgeErrorResponse.schema.json)
+
+### Example
+
+Response message from a Desktop Agent:
+
+```json
+// agent-B -> DAB
+{
+    "type":  "findInstancesResponse",
+    "payload": {
+        "appIdentifiers":  [
+            { "appId": "myApp", "instanceId": "4bf39be1-a25b-4ad5-8dbc-ce37b436a344"},
+            { "appId": "myApp", "instanceId": "4f10abb7-4df4-4fc6-8813-bbf0dc1b393d"},
+        ]
+    },
+    "meta": {
+        "requestUuid": "<requestUuid>",
+        "responseUuid":  "<responseUuidAgentB>",
+        "timestamp":  "2020-03-...",
+    }
+}
+```
+
+The bridge receives and collates the responses, augmenting each appIdentifier with a `desktopAgent` field, producing the following collated response which it sends back to agent-A:
+
+```json
+// DAB -> agent-A
+{
+    "type":  "findInstancesResponse",
+    "payload": {
+        "appIdentifiers":  [
+            { "appId": "myApp", "instanceId": "4bf39be1-a25b-4ad5-8dbc-ce37b436a344", "desktopAgent": "agent-B"},
+            //"desktopAgent" added by DAB
+            { "appId": "myApp", "instanceId": "4f10abb7-4df4-4fc6-8813-bbf0dc1b393d", "desktopAgent": "agent-B" },
+            { "appId": "myApp", "instanceId": "920b74f7-1fef-4076-adef-63b82bae0dd9", "desktopAgent": "agent-C" },
+        ]
+    },
+    "meta": {
+        "requestUuid": "<requestUuid>",
+        "responseUuid":  "<responseUuidDAB>", 
+        "timestamp":  "2020-03-...",
+        "sources": [ //added by DAB
+            { "desktopAgent": "agent-A" },
+            { "desktopAgent": "agent-B" },
+        ]
+    }
+}
+```
+
+:::info
+If a target Desktop Agent was specified in the request (via a `desktopAgent` field in `payload.app`), then the DAB is not collating responses and does not need to generate a unique `meta.responseUuid` and can quote that given by the responding Desktop Agent.
```

Comment:
Good point. To avoid confusion lets make that a MUST
```suggestion
If a target Desktop Agent was specified in the request (via a `desktopAgent` field in `payload.app`), then the DAB is not collating responses and does not need to generate a unique `meta.responseUuid` and MUST quote that given by the responding Desktop Agent.
```